### ISSUE TITLE
Cut EA JDK?

### ISSUE THREAD

**Issue by boner-cmd** [boner-cmd/JLinkMC#35]
(Affects #22 priority)

Since the target use case (PaperMC) can't use it or test it and the workflow to set it up is so different, should functionality and support for the early access JDK be dropped entirely? While it would be useful to jlink a muslc-compatible JDK and skip a lot of steps, it feels like hopefully that functionality will just come over to GA someday, at which point the entire workflow will need to change accordingly. However, someday isn't now, and duplicating the effort by developing in parallel in anticipation for a future migration doesn't offer much now.

On the other hand, does not using the EA create a self-fulfilling prophecy when it isn't used enough to hit GA?

Also, if the EA muslc build is removed and the version number incremented when the next build hits GA, does that mean that it could be run as though it were current?

i.e. Program X only uses the current GA version of Java, which is arbitrarily 44. The EA, 45, has a muslc build. When 45 hits GA, the muslc build is removed and incremented to 46, remaining in EA. However, if you still have a download from when the muslc build was numbered 45 (optimally, the last build before 45 became the GA release), could Program X accept the "old EA" muslc build since the version number hadn't been updated yet?

That logic is kind of dangerously close to #19 but maybe that means there was merit to the idea...

**Comment by boner-cmd** [comment]
This road is leading toward things like "compile Portola JDK from source" so the EA definitely needs to go on the back burner while everything else is hammered out. That also means to avoid errors, it should get cut from production until it's ready to get re-branched and take the attention it deserves.

**Comment by boner-cmd** [comment]
This shouldn't get closed until the pull is merged
